db/DB.js

```diff
@@ -212,6 +212,19 @@ const DB = {
     return updatedSubscriber;
   },
 
+  async setBreachesLastShownNow(subscriber) {
+    const nowDateTime = new Date(Date.now());
```

Comment:
I believe `new Date()` returns the current date, so you shouldn't need the `Date.now()`.